Resources/Textures/Clothing/Hands/Gloves/janitor.rsi/meta.json

```
@@ -1,7 +1,8 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Drawn by Ubaser, recolored by InfinityRedPanda.",
+  "copyright": "Drawn by Ubaser, resprited by mishutka09 (1152277579206774854 on discord.",
```

Biome (2.1.2)

[error] The key copyright was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)